arcadia-users-group/20220822_intro_to_markdown_syntax.md

```diff
@@ -0,0 +1,155 @@
+# Introduction to markdown syntax
```

Comment:
Is this in a folder named `arcadia-users-group` because you think down the road there might be different trainings outside of the users group that are then in different folders? I also think down the line that within `arcadia-users-group` there should be subfolders with the materials in them since you could imagine that for a given lesson/training there would be multiple files such as example scripts in addition to the landing markdown page.

Reply:
yes to the first part -- for example, I plan to have maybe a workshops folder, for content from the git and github workshop that will be happening.

and yes, if think for lessons that have multiple parts, those could be folders

Reply:
👍 on having folders per week (or training session) to organize multiple files together as needed

Reply:
kk i'll implement this switch after today's session